en/docs/api-gateway/next/deployment/deployment-modes/kubernetes/kubernetes-standalone.md

```
@@ -237,12 +237,12 @@ helm install ap-gateway oci://ghcr.io/wso2/api-platform/helm-charts/gateway \
 ### Port-forward Gateway Controller Service
 
 ```bash
-kubectl port-forward svc/ap-gateway-controller 9090:9090
+kubectl port-forward svc/ap-gateway-controller 9090:9090 9094:9094
 ```
 
 ### Verify gateway controller admin endpoint is running
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)